test/suites/logscan/log-scan.robot

```diff
@@ -27,34 +81,12 @@ Should Not Find Cannot Patch Resource
 
 Services Should Not Timeout When Stopping
     [Documentation]    Logs should not say "Timed out waiting for services to stop"
-    [Tags]    robot:exclude
     Pattern Should Not Appear In Log Output    ${CURSOR}    Timed out waiting for services to stop
```

Comment:
We no longer have this exact string, it was changed to `MICROSHIFT STOP TIMED OUT`. Shall we change it in this PR?